docs/styling.md

```diff
@@ -0,0 +1,119 @@
+# Styling in Store Framework Jamstack
+
+Styling in Store Framework Jamstack leverages the library [theme-ui](https://theme-ui.com/). By using this library we have a pre-defined way of creating themes that can be shared across components on the store.
+
+Theme UI already has a [gatsby plugin](https://www.gatsbyjs.com/docs/how-to/styling/theme-ui/) available. This plugin, however, was not attending our needs because we want to optimize how CSS is merged at runtime, and this plugin merges the CSS at build time.
```

Comment:
I don't think that we need to say it here, justifying the choices. What about explaing your plugin? 
Reduce lines by using: 

"The Store Framework Jamstack  [custom plugin](https://github.com/vtex/faststore/tree/master/packages/gatsby-plugin-theme-ui) optimizes how CSS is merged at runtime."